### ISSUE TITLE
What is the hash that is specified? 

### ISSUE THREAD

**Issue by spuder** [lynx44/windows_ssl#4]
What value needs to be set for the hash. 

Searching the internet, it seems that this is the same as the 'thumbprint' ? Is that correct? 

https://msdn.microsoft.com/en-us/library/ms734695(v=vs.110).aspx

**Comment by lynx44** [comment]
Yes, it looks like that's the same value (although with no spaces). You can also view it once it's been imported in IIS under IIS Root->Server Certificates, where the value will be displayed in the Certificate Hash column.

For more information, see the MS documentation for netsh add cert - certhash argument:

https://msdn.microsoft.com/en-us/library/windows/desktop/cc307220(v=vs.85).aspx
